doc/library/gpuarray/fft.txt

```diff
@@ -0,0 +1,44 @@
+.. _libdoc_gpuarray_fft:
+
+==============================================
+:mod:`gpuarray.fft` -- Fast Fourier Transforms
+==============================================
+
+Performs Fast Fourier Transforms (FFT) on the GPU.
+
+FFT gradients are implemented as the opposite Fourier transform of the output gradients.
+
+.. warning ::
+    The real and imaginary parts of the Fourier domain arrays are stored as a pair of float32
+    array, emulating complex64. Since theano does not support complex
+    number operations, care must be taken to manually implement operators such as complex
```

Comment:
Theano support them in limited way. But we don't support the grad on it. So maybe tell that we don't support grad on complex?